Fix typo null as the bottom of proto chain (#14033)
mdn/content: files/en-us/web/javascript/inheritance_and_the_prototype_chain/index.md

When it comes to inheritance, JavaScript only has one construct: objects. Each object has a private property which holds a link to another object called its **prototype**. That prototype object has a prototype of its own, and so on until an object is reached with `null` as its prototype. By definition, `null` has no prototype, and acts as the final link in this **prototype chain**.

> Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which has `null` as its prototype.

Reviewer: ```suggestion
Nearly all objects in JavaScript are instances of {{jsxref("Object")}}, which has `null` as its prototype.
```

> Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which sits just above `null` on the top of a prototype chain.

Reviewer 2: I agree that the notion of _top_ is not good, but then we can't speak of below and above.

What about:
```suggestion
Nearly all objects in JavaScript are instances of {{jsxref("Object")}} which has `null` as its prototype.
```
